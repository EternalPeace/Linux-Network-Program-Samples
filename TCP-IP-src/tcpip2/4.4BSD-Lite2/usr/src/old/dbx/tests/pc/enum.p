program enumtest (input, output);
type
    Color = (RED, GREEN, BLUE);

var c1, c2 : Color;

procedure P (c : Color);
begin
    writeln('c = ', ord(c):1);
end;

begin
    c1 := BLUE;
    c2 := GREEN;
    P(RED);
end.
