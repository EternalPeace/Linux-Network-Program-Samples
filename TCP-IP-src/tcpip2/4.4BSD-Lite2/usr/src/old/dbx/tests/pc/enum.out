reading symbolic information ...
type Color = (RED, GREEN, BLUE);
var c1 : Color;
[1] stop in P
[1] stopped in P at line 8
    8   begin
P(c = RED), line 8 in "enum.p"
program(), line 15 in "enum.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0x105
RED 
BLUE GREEN 
